docs/examples/example_merge_species.py

```diff
@@ -0,0 +1,77 @@
+from pyrokinetics import Pyro, template_dir
+from pint import UnitRegistry
+import numpy as np
+
+# data path
+spr_path = '/home/ab1209/pyrokinetics/SPR-JETTO/qmin_264_imp'
```

Comment:
This might work better as an executable example where we provide relevant equilibrium and profile files as with tests/other examples so users can expect the example script to work unlike here where you've hard-coded a path from your local system.

Reply:
If this could reuse existing example files then that might be even nicer (and definitely shouldn't add the files that you're currently referencing!)

Reply:
Yes agreed about using existing template files